### ISSUE TITLE
QR code not recognized

### ISSUE THREAD

**Issue by lafriks** [nimiq/qr-scanner#124]
I don't know why but this QR code does not seems to be recognized when scanning from file while is recognized when scanned using camera:

https://raw.githubusercontent.com/eu-digital-green-certificates/dgc-testdata/main/LT/png/2.png

**Comment by sisou** [comment]
When uploaded at https://nimiq.github.io/qr-scanner/demo/ the image you linked is decoded fine. This is on Chrome and Firefox on Linux. What browser on which OS are you using?

**Comment by lafriks** [comment]
Yes, sorry pasted wrong link https://raw.githubusercontent.com/eu-digital-green-certificates/dgc-testdata/main/SG/png/3.png

**Comment by danimoh** [comment]
For me this scans just fine now, both via the native `BarcodeDetector` on Android and the worker engine on my laptop.
Please test again with the newest version.
